### PR TITLE
[#329] Bump version to 0.4.0

### PR DESCRIPTION
**Pull request by korydraughn**
Reset the package revision to 0 too.

All tests pass.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,10 +11,10 @@ include(RequireOutOfSourceBuild)
 
 set(IRODS_HTTP_API_BINARY_NAME "irods_http_api")
 
-set(IRODS_CLIENT_VERSION "0.3.0")
+set(IRODS_CLIENT_VERSION "0.4.0")
 set(IRODS_HTTP_API_VERSION "${IRODS_CLIENT_VERSION}")
 
-set(IRODS_PACKAGE_REVISION "1")
+set(IRODS_PACKAGE_REVISION "0")
```
Comment (trel) [inline CMakeLists.txt:17]: is this right?  it's always been 1.

Comment (korydraughn) [reply CMakeLists.txt:17]: Based on our other projects, yes.

Comment (trel) [reply CMakeLists.txt:17]: okie doke.

**Review by alanking — APPROVED** [review]
\# it

**Comment by korydraughn** [comment]
Pounded.

**Pull request merged**